packages/legacy/core/App/modules/history/ui/components/HistoryListItem.tsx

```diff
@@ -187,7 +186,7 @@ const HistoryListItem: React.FC<Props> = ({ item }) => {
             {renderCardDate(item.content.createdAt)}
           </View>
           <View style={styles.arrowContainer}>
-            <Assets.svg.iconChevronRight />
+            <Assets.svg.iconChevronRight {...{ color: ColorPallet.brand.primary }} />
```

Comment:
Just curious, why the spread instead of just `color={ColorPallet.brand.primary}`?